src/components/RegistrationForm/Form/Form.jsx

```diff
@@ -1,30 +1,122 @@
-import {FormField, FormFieldPassword} from './fields/index.js';
-import {StyledForm, StyledButton} from './Form.styled';
+import { useForm } from 'react-hook-form';
 
+import { useAxiosPost } from '../../../hooks/useAxiosPost.js';
+
+import RegisterTerms from './RegisterTerms/RegisterTerms.jsx';
+import { FormField, FormFieldPassword } from './fields/index.js';
+import { FieldValidation } from './FieldValidation/FieldValidation.jsx';
+import { StyledForm, StyledButton } from './Form.styled';
+
+const BASE_API_URL = 'https://marketplace-p93k.onrender.com/api';
```

Comment:
That URL should be removed. It already created and use from axios API file..

Reply:
I use useAxiosPost() with parameter "url", and  BASE_API_URL it is argument for useAxiosPost()